src/savers/tvg/tvgTvgSaver.cpp

```diff
@@ -763,6 +760,9 @@ bool TvgSaver::save(Paint* paint, const string& path, bool compress)
         return false;
     }
 
+    this->path = strdup(path.c_str());
```

Comment:
any reason to change this?

Reply:
if it stays where it was and the if condition in l.761 is true, who will released this memory?

Reply:
ok, good catch thanks.